app/src/main/res/raw/about_demo_custom_item.html

```diff
@@ -2,7 +2,7 @@
 <html>
     <body>
         <p>
-            Here is the file associated to the custom menu item clicked.
+            Here is the file associated to the legal information menu item clicked.
```

Comment:
It seems that content of `about_demo_custom_item.xml` and `about_demo_legal_information_item.html` are inverted.